build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+set -e
+
+# Install programs required for build
+apk --no-cache add nodejs-lts git
+npm install -g yarn bower
```

Comment:
Correct, I wholeheartedly agree, but since we are using alpine here, there is no apt-get. Apk is the closest solution, but that has no yarn installer. This means that the install script is the only other solution. but since that one was not working when I tried last time, I have just used this solution.